src/NServiceBus.Core/InitializableEndpoint.cs

```diff
@@ -182,5 +183,6 @@ string GetInstallationUserName()
         PipelineConfiguration pipelineConfiguration;
         PipelineSettings pipelineSettings;
         SettingsHolder settings;
+        CriticalError criticalError;
```

Comment:
There is a whole bunch of field access going on in this class. So I decided to stick to that and align with the consistently bad approach ;)